etc/.golangci.yaml

```diff
@@ -46,6 +46,7 @@ linters:
     - varnamelen
     - wrapcheck
     - wsl
+    - gosec
```

Comment:
Don’t disable all of gosec, just the one or two codes that you have to have for the implementation. rdk+other modules with linters should have an example